libgoal/participation.go

```diff
@@ -152,6 +106,14 @@ func (c *Client) GenParticipationKeysTo(address string, firstValid, lastValid, k
 		return
 	}
 
+	// If the key is being installed, remove it afterwards.
+	if install {
+		// Explicitly ignore any errors
+		defer func(name string) {
+			_ = os.Remove(name)
+		}(partKeyPath)
```

Comment:
Did `defer os.Remove(partKeyPath)` cause the static analysis tool to warn?

Reply:
It made my IDE warn, I figured it was best practice.  If it's bad form I can remove it.